src/encoded/static/components/search.js

```diff
@@ -1,111 +1,111 @@
-'use strict';
-var React = require('react');
-var cloneWithProps = require('react/lib/cloneWithProps');
-var queryString = require('query-string');
-var button = require('../libs/bootstrap/button');
-var {Modal, ModalHeader, ModalBody, ModalFooter} = require('../libs/bootstrap/modal');
-var dropdownMenu = require('../libs/bootstrap/dropdown-menu');
-var svgIcon = require('../libs/svg-icons').svgIcon;
-var cx = require('react/lib/cx');
-var url = require('url');
-var _ = require('underscore');
-var globals = require('./globals');
-var image = require('./image');
-var search = module.exports;
-var { donorDiversity, BrowserSelector } = require('./objectutils');
-var dbxref = require('./dbxref');
-var audit = require('./audit');
-var objectutils = require('./objectutils');
-var {BiosampleSummaryString, BiosampleOrganismNames} = require('./typeutils');
-
-var DbxrefList = dbxref.DbxrefList;
-var statusOrder = globals.statusOrder;
-var SingleTreatment = objectutils.SingleTreatment;
-var AuditIndicators = audit.AuditIndicators;
-var AuditDetail = audit.AuditDetail;
-var AuditMixin = audit.AuditMixin;
-var DropdownButton = button.DropdownButton;
-var DropdownMenu = dropdownMenu.DropdownMenu;
+import React from 'react';
```

Comment:
I wouldn’t mind moving the type-specific search rendering functions to their own files (Experiment search result component to experiment.js) as we do with the newer object search results.